editor/src/components/canvas/move-template.spec.browser.tsx

```diff
@@ -525,11 +525,11 @@ describe('moveTemplate', () => {
       makeTestProjectCodeWithSnippet(`
         <View style={{ ...props.style }} data-uid='aaa'>
           <View
-            style={{ backgroundColor: '#0091FFAA', position: 'relative', left: 52, top: 61, width: 256, height: 202 }}
+            style={{ backgroundColor: '#0091FFAA', position: 'absolute', left: 52, top: 61, width: 256, height: 202 }}
             data-uid='bbb'
           >
             <View data-uid='ccc'>
-              <View data-uid='ddd' />
+              <View data-uid='ddd'  style={{ backgroundColor: '#0091FFAA', position: 'absolute', left: 10, top: 10, width: 100, height: 100 }} />
```

Comment:
This looks suspicious. Why have these tests changed?

Reply:
Oh was this the general problematic tests that were occasionally failing on master?